pyomo/contrib/pynumero/doc/source/index.rst

```diff
@@ -0,0 +1,32 @@
+PyNumero Documentation
+=========================
+
+.. toctree::
+   :maxdepth: 1
+
+   overview
+   installation
+   examples
+   API documentation <apidoc/pynumero>
+   license
+   developers
+
+Sandia Funding Statement
+--------------------------------
+
+Sandia National Laboratories is a multimission laboratory managed and operated by National Technology and 
+Engineering Solutions of Sandia.
+
+IDEAS Funding
```

Comment:
@jsiirola, same question here about the funding statement

@santiagoropb IDAES is misspelled.

Reply:
fixed misspelling